lib/utils.js

```diff
@@ -221,10 +221,18 @@ const xmlToJson = (xml) => {
 function returnTransports(transportOptions) {
   // eslint-disable-next-line global-require
   const { Connection } = require('./Connection');
-  const availableTransports = ['idb', 'rest'];
+  // eslint-disable-next-line global-require
+  const { readFileSync } = require('fs');
+  const availableTransports = ['idb', 'rest', 'ssh'];
   const transports = [];
 
+  if (transportOptions.privateKey) {
+    // eslint-disable-next-line no-param-reassign
+    transportOptions.privateKey = readFileSync(transportOptions.privateKey, 'utf8');
+  }
+
```

Comment:
I thought all this was going to be left to ssh2 to handle.

Reply:
In here this is done for our automated tests, the read in private key is passed to ssh2 to handle.

Users would have code like the following:

```
const { readFileSync } = require('fs');

const privateKey = readFileSync('path/to/privateKey', 'utf-8');

// NOTE if your privateKey also requires a passphrase provide it

const connection = new Connection({
  transport: 'ssh',
  transportOptions: { host: 'myhost', username: 'myuser', privateKey, passphrase: 'myphrase' }
});

```

Reply:
Couldn't the tests do that themselves?

Reply:
True reading in the PK can be done within the individual test files `CommandCallFunctional`, `SqlCallFunctional`, `ProgramCallFunctional`.